dap-java.el

```diff
@@ -93,30 +93,30 @@ If the port is taken, DAP will try the next port."
   "Get class FDQN."
   (-if-let* ((symbols (lsp--get-document-symbols))
              (package-name (-some->> symbols
-                             (-first (-lambda ((&hash "kind")) (= kind 4)))
-                             (gethash "name")))
+                             (-first (-lambda ((&DocumentSymbol :kind)) (= kind 4)))
```

Comment:
it is a good time to replace those with the corresponding constants.

Reply:
I forgot we already have constants for those, I'll replace them

Reply:
Done!